src/Qualys/functions/public/Invoke-QualysRestCall.ps1

```diff
@@ -64,6 +64,7 @@ function Invoke-QualysRestCall {
         }
 
         Invoke-RestMethod @IVRSplat
+        $Script:APICallCount++
```

Comment:
can't do a warning when it approaches the limit because the limit is time sensitive and other scripts or just me doing something may be using the api in the bg using that count ):